src/Project/ProjectVersionGit.php

```diff
@@ -126,6 +134,7 @@ protected function queryGitData()
         }
         $data['url'] = strtr($url, array('.git' => '', 'ssh://' => $R, 'git://' => $R, 'ftp://' => $R, 'ftps://' => $R, 'rsync://' => $R));
         $data['tag'] = exec('git describe --tags --always');
+        $data['closestTag'] = exec('git describe --tags --always --abbrev=0');
```

Comment:
I think it's fine to return the hash then. The consumer of the functionality needs to make sure something reasonable happens with the returned value...